flow/Stats.actor.cpp

```diff
@@ -45,21 +45,32 @@ double Counter::getRate() const {
 }
 
 double Counter::getRoughness() const {
-	double elapsed = now() - interval_start;
+	double elapsed = now() - roughness_interval_start;
 	if(elapsed == 0) {
 		return 0;
 	}
 
+	// If we have time samples t in T, and let:
+	// n = size(T) = interval_delta
+	// m = mean(T) = elapsed / interval_delta
+	// v = sum(t^2) for t in T = interval_sq_time
+	// 
+	// The formula below is: (v/(m*n)) / m - 1
+	// This is equivalent to (v/n - m^2) / m^2 = Variance(T)/m^2
+	// Variance(T)/m^2 is equal to Variance(t/m) for t in T
 	double delay = interval_sq_time / elapsed;
-	return delay * getRate() * 2;
```

Comment:
I couldn't really explain this factor of 2, as it seemingly caused the roughness values to differ from what was documented.